packages/flutter/README.md

```diff
@@ -7,110 +7,3 @@ focused on low-latency input and high frame rates on Android and iOS.
 
 See the [getting started guide](https://flutter.io/getting-started/) for
 information about using Flutter.
-
-## How To Write International Flutter Applications
-
-If your app might be deployed to users who speak another language you'll need to "internationalize" it. That means you'll need to write the app in a way that makes it possible to "localize" things like text and layouts for each language or "locale" that the app supports. Flutter provides widgets and classes that help with internationalization and the Flutter libraries themselves are internationalized.
-
-The [Locale] class is used to identify the user's language. Mobile devices support setting the locale for all applications, usually via a system settings menu. Internationalized apps respond by redisplaying values that are locale-specific. For example if the user switches the device's locale from English to French then a Text widget that displayed "CANCEL" would be rebuilt with "ANNULER".
-
-The [Localizations] widget defines the locale for its child and the localized resources that the child depends on. The `WidgetsApp` widget creates a Localizations widget and rebuilds it if the system's locale changes.
-
-You can always lookup an app's current locale with `Localizations.localeOf()`:
-```
-Locale myLocale = Localizations.of(context);
-```
-
-The [Localizations] widget is used to load and lookup localized values. Apps refer to the Localizations widget with `Localizations.of(context, type)`. If the device's locale changes, the Localizations widget automatically loads values for the new locale and then rebuilds widgets that used it to look up localized values. This happens because Localizations is effectively an [InheritedWidget]. When a build function refers to an inherited widget an implicit dependency on the inherited widget is created. When an inherited widget changes, its dependent contexts are rebuilt.
```

Comment:
I'd remove "effectively" here: it either implements the API, or it doesn't: the details aren't relevant to the user.

Reply:
I used "effectively" because it behaves like a inherited widget vis dependencies and it's used like an InheritedWidget would be. The widget class actually doesn't implement InheritedWidget because it's stateful. And I guess I used "effectively" because I didn't want to explain all that :-)

Reply:
Maybe use "similar to" then?  Something like "This happens because Localizations is similar to an [InheritedWidget] in how it propagates information, but is stateless."